functions.php

```diff
@@ -34,6 +34,16 @@ function standards_example_function( $color ) {
 }
 
 /**
+ * Saves a meta value on post save.
+ *
+ * @param int $post_id The post ID.
+ */
+function sanitization_example_function( $post_id ) {
+	update_post_meta( $post_id, 'some_key', $_POST['some_key'] );
```

Comment:
Missing wp_unslash and sanitization of the input value.

Should also check if the array key is set before using it.